internal/view/exec.go

```diff
@@ -337,7 +337,8 @@ func sshIn(a *App, fqn, co string) error {
 		return fmt.Errorf("os detect failed: %w", err)
 	}
 
-	args := buildShellArgs("exec", fqn, co, a.Conn().Config().Flags().KubeConfig)
+	flags := a.Conn().Config().Flags()
+	args := buildShellArgs("exec", fqn, co, flags.KubeConfig, flags.Context, flags.BearerToken)
```

Comment:
@fabianonunes I think it will be best to just pass in the k8s flags at the call site and let `buildShellArgs` figure out what it need to extract out.